conf/core/addressTagKeys

```diff
@@ -6,11 +6,8 @@
 #
 # The format is: <address component type>=<tag key 1>,<tag key 2>... Keep address component types and tag keys in alphabetical order.
 
-block=addr:block,addr:block_number
 full_address=address,addr:full,ref:addr,source:addr,source:address
-number=addr:conscriptionnumber,addr:door,addr:housenumber,addr:provisionalnumber,addr:street:sym_ul,addr:streetnumber
+house_number=addr:conscriptionnumber,addr:door,addr:housenumber,addr:provisionalnumber,addr:street:sym_ul,addr:streetnumber
 street=addr:street,addr:street:name
-street_corner=addr:street:corner
 street_prefix=addr:street:prefix
-street_side=addr:side
 street_suffix=addr:street:suffix,addr:street:type
```

Comment:
This file lets you map OSM address tag keys to different types of addresses.  I used TagInfo as my source.